Apache-Arrow-Flight-Server-Tester/main.py

```diff
@@ -5,10 +5,56 @@
 
 import pyarrow
 from pyarrow import flight
+from pyarrow._flight import FlightClient
+
+
+# PyArrow Functions.
+def list_flights(flight_client: FlightClient) -> None:
+    response = flight_client.list_flights()
+
+    print(list(response))
+
+
+def get_schema(flight_client: FlightClient, table_name: str) -> None:
+    upload_descriptor = flight.FlightDescriptor.for_path(table_name)
+    response = flight_client.get_schema(upload_descriptor)
+
+    print(response.schema)
+
+
+def do_get(flight_client: FlightClient, query: str) -> None:
+    ticket = flight.Ticket(query)
+    response = flight_client.do_get(ticket)
+
+    for batch in response:
+        pprint.pprint(batch.data.to_pydict())
+
+
+def do_put(flight_client: FlightClient, table_name: str) -> None:
+    upload_descriptor = flight.FlightDescriptor.for_path(table_name)
+    writer, _ = flight_client.do_put(upload_descriptor, get_pyarrow_schema())
+
+    record_batch = create_record_batch(10000)
```

Comment:
Maybe add a comment that describes what the magic number of 10,000 means.

Reply:
Since its just a number that is changed all the time during testing I moved it up to an argument so it feels less like a magic number.